app/javascript/dashboard/components/widgets/conversation/ReplyBox.vue

```diff
@@ -211,18 +211,19 @@ export default {
       if (this.message.length > this.maxLength) {
         return;
       }
+      const newMessage = this.message;
+      this.clearMessage();
```

Comment:
This should be inside the check for canned response list. Otherwise `/` commands for canned response will not work.
<img width="532" alt="Screenshot 2020-06-28 at 6 55 31 PM" src="https://user-images.githubusercontent.com/2246121/85948883-f2405700-b970-11ea-8fbe-be1a9acab92c.png">